packages/braid-design-system/src/lib/components/Tabs/TabPanel.tsx

```diff
@@ -52,18 +51,10 @@ export const TabPanel = ({
       {...a11y.tabPanelProps({ panelIndex, isSelected })}
       display={isSelected ? undefined : 'none'}
       position="relative"
-      outline="none"
       className={styles.tabPanel}
       {...buildDataAttributes({ data, validateRestProps: restProps })}
     >
       {isSelected || renderInactive ? children : undefined}
-      <Overlay
-        zIndex={1}
-        boxShadow="outlineFocus"
-        borderRadius="large"
```

Comment:
This `borderRadius` needs to be transferred to the parent panel element above. TabPanels are looking pointy 😄